storm-core/src/jvm/org/apache/storm/utils/IPredicate.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.storm.utils;
+
+/**
+ * Created by rfarivar on 1/12/16.
+ */
+public interface IPredicate {
```

Comment:
Have the javadocs explain the interface, not who created it and when, git is for that.